serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -181,9 +181,33 @@ export class _GLTFMaterialExporter {
             }
         }
 
+        // Do a naive check for materials that contain extensions that have texture-related members.
+        if (material.extensions) {
+            for (let extension in material.extensions){
+                let extensionObject = material.extensions[extension];
+                for (let member in extensionObject){
+                    if (member.indexOf("Texture") !== -1){
+                        if (extensionObject[member]){
+                            return true;
+                        }
+                    }
+                }
+            }
+        }

```

Comment:
Hmm, are you sure you want to leave it like this? It seems better if the code asks the extension code whether there are textures or not?

Reply:
Yeah, I considered that, as well ^^^ https://github.com/BabylonJS/Babylon.js/pull/10237#discussion_r617731942 as a mapping. Most likely this can be done as an interface inherited by the other material extension interfaces.

Reply:
Ended up adding a base class to our glTF interface, now when we initialize these serialized members, we'll define a function. I don't think this function gets serialized by default, but I'll check.